src/components/Page/RichText.vue

```diff
@@ -85,6 +92,7 @@ export default {
 			'currentPageDirectory',
 			'currentPageFilePath',
 			'isPublic',
+			'loading',
```

Comment:
Seems to be not introduced here but would be nice to have the naming a bit more clear to tell what loading state this is. Not blocking.

Reply:
It's a generic object holding the loading state for anything in Collectives, see https://github.com/nextcloud/collectives/blob/c391ee9766cc0f7b67381722f1e49e8c5fba995e/src/store/store.js#L30 and https://github.com/nextcloud/collectives/blob/c391ee9766cc0f7b67381722f1e49e8c5fba995e/src/store/store.js#L56-L57

Reply:
All right, thanks for the insight :)